scannercli.properties

```diff
@@ -16,6 +16,15 @@ flavors.macos.label=Mac OS X 64-bit
 flavors.any.label=Any (Requires a pre-installed JVM)
 flavors.docker.label=Docker
 
+6.0.description=New bootstrapping mechanism and JRE provisioning with SQ 10.6+ and SonarCloud
+6.0.date=2024-06-04
+6.0.changelogUrl=https://sonarsource.atlassian.net/issues/?jql=project%20%3D%2010143%20AND%20fixVersion%20%3D%2015344
+6.0.downloadUrl.linux=https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-6.0.0.4432-linux.zip
+6.0.downloadUrl.win=https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-6.0.0.4432-windows.zip
+6.0.downloadUrl.macos=https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-6.0.0.4432-macosx.zip
+6.0.downloadUrl.any=https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-6.0.0.4432.zip
+6.0.downloadUrl.docker=https://hub.docker.com/r/sonarsource/sonar-scanner-cli
```

Comment:
Note that the docker image is still containing the scanner 5.0.1, so we will need to do a sprint.